packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -26,8 +26,10 @@
     "@salesforce/salesforcedx-utils-vscode": "43.14.0",
     "ajv": "^6.1.1",
     "eslint": "4.16.0",
-    "eslint-plugin-lwc": "0.4.0",
-    "lwc-language-server": "1.6.6",
+    "eslint-plugin-lwc":
+      "http://npm.lwcjs.org/eslint-plugin-lwc/-/eslint-plugin-lwc-0.4.0/643f5e18247ca8c0a60bf972f4d3795c97e8e6fe.tgz",
```

Comment:
@rsalvador - Due to possibly, https://github.com/jdxcode/npm-register/issues/129, the npm.lwcjs.org registry is not properly sending back the packages for @babel and @types. Thus, instead of using the package that way, we embed the tar.url.

This was done by getting the dist.tarball using, e.g., `npm view --registry=https://npm.lwcjs.org/ lwc-language-server@1.6.1`. The tar.gz URL option is supported through https://docs.npmjs.com/files/package.json#urls-as-dependencies

Reply:
When you are back from the weekend, it might be worthwhile to see if we want to use https://docs.npmjs.com/files/package.json#bundleddependencies to the lwc-language-server so that we can do it without specifying all its dependencies chain. I think this is a consequence of the refactor that was going on. lwc-language-server@1.5.1 didn't need all these other dependencies brought in.

Reply:
Ok, will try that when publishing the next lwc-language-server version